### ISSUE TITLE
deploy:normalize_assets, 'No such file or directory'

### ISSUE THREAD

**Issue by rckv** [capistrano/rails#184]
I use `capistrano-rails (1.1.7)` on OS X 10.11.6 deploying my Rails app to a Ubuntu server (14.04.4 LTS).

In `config/deploy.rb`, I set `normalize_asset_timestamps` with an array
```
set :normalize_asset_timestamps, %w{public/assets public/images}
```

When I run `cap deploy:normalize_assets`, I get the following warning
```
00:00 deploy:normalize_assets
      01 find ["public/assets,", "public/images"] -exec touch -t 201609080234.52 {} ';'; true
      01 find: `[public/assets,,': No such file or directory
      01 find: `public/images]': No such file or directory
```

I run `find public/assets public/images -exec touch -t 201609080234.52 {} ';'; true` on the Ubuntu server and it succeeds.

It seems to be caused by string interpolation with Array
```
# lib/capistrano/tasks/assets.rake#L15
execute :find, "#{assets} -exec touch -t #{asset_timestamp} {} ';'; true"
```

**Comment by will-in-wi** [comment]
As an experiment, could you try

```ruby
set :normalize_asset_timestamps, %w{public/assets, public/images}.join(' ')
```

**Comment by rckv** [comment]
@will-in-wi Yes, it works. 
```
# config/deploy.rb
set :normalize_asset_timestamps, %w{public/assets public/images}.join(' ')
```
The join method returns a string `"public/assets public/images"` so it works well with string interpolation.
```
00:00 deploy:normalize_assets
      01 find public/assets public/images -exec touch -t 201609080445.58 {} ';'; true
```

**Comment by mattbrictson** [comment]
Fixed by #185
